### PR TITLE
i18n(id): translate `guides/css-and-tailwind.mdx`

### PR DESCRIPTION
**Pull request by oggnimodd**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- Translate `guides/css-and-tailwind.mdx` to Indonesian

### CODE REVIEW

File: docs/src/content/docs/id/guides/css-and-tailwind.mdx [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Presets',
+			ocean: 'Ocean',
+			forest: 'Forest',
+			oxide: 'Oxide',
+			nebula: 'Nebula',
+			default: 'Default',
+			random: 'Random',
```
Comment (delucis) [inline docs/src/content/docs/id/guides/css-and-tailwind.mdx:259]: Noticing that these haven’t been translated. Was that intentional? In other languages we generally have translated these.

**Review by delucis — COMMENTED** [review]
Thank you for the translation @oggnimodd! Just noticed one small thing, but otherwise looks great 🙌

File: docs/src/content/docs/id/guides/css-and-tailwind.mdx (hunk 1 above)
Comment (oggnimodd) [reply docs/src/content/docs/id/guides/css-and-tailwind.mdx:259]: Thanks for catching that, I missed those

**Review by delucis — APPROVED** [review]
Great — thank you for making that change. Ready to launch 🚀

**Pull request merged**